src/fidesops/service/connectors/saas_connector.py

```diff
@@ -142,6 +135,15 @@ def mask_data(
         rows: List[Row],
     ) -> int:
         """Execute a masking request. Return the number of rows that have been updated"""
+        query_config = self.query_config(node)
+        prepared_requests = [
+            query_config.generate_update_stmt(row, policy, request) for row in rows
+        ]
+        rows_updated = 0
+        for prepared_request in prepared_requests:
+            self.client().send(prepared_request)
```

Comment:
What happens if you get a non-200 response from this request?

Reply:
Going to fix this in #182

Reply:
Fixed with changes merged in from main branch.

Reply:
thanks for the update!